src/main/scala/io/opentargets/etl/backend/target/GeneWithLocation.scala

```diff
@@ -42,6 +42,6 @@ object GeneWithLocation extends LazyLogging {
         "locations",
         safeArrayUnion(col("HPA_main"), col("HPA_additional"), col("HPA_extracellular_location")))
 
-    hpaDf.as[GeneWithLocation]
+    hpaDf.select(col("id"), col("locations")).as[GeneWithLocation]
```

Comment:
@JarrodBaker, based on the comment I made, the correct code here could be this for the intention above

```Scala
hpaDf.as[GeneWithLocation].map(identity)
```